src/charm.py

```diff
@@ -668,11 +176,13 @@ def _report_service_ips(self) -> List[str]:
         """
         api = self._core_v1_api()
         services = api.list_namespaced_service(  # type: ignore[attr-defined]
-            namespace=self._namespace
+            namespace=self._namespace, label_selector=self._label_selector
```

Comment:
Potentially remove in the future, leave for now

Reply:
Removed, thanks!